_data/glossary/en/signature-script.yaml

```diff
@@ -8,8 +8,8 @@ required:
 
     summary_max_255_characters_no_formatting: >
         Data generated by a spender which is almost always used as
-        variables to satisfy a pubkey script. Called a scriptSig in
-        code.
+        variables to satisfy a pubkey script. Signature Scripts are called
+        `scriptSig` in code.
```

Comment:
I don't think this change is necessary, but I'm ok with it being made provided the backtics are removed; please see https://github.com/bitcoin-dot-org/bitcoin.org/pull/1323/files#r69164403 for details.